client/src/components/related_products/Lists/Outfit.jsx

```diff
@@ -41,19 +43,24 @@ function Outfit() {
 
   useEffect(() => {
     localStorage.setItem('outfit', JSON.stringify(outfit));
+    if (outfit.length > 3) {
+      setMaxWidth((outfit.length - 3) * 220 + (220 - 2));
+    } else {
+      setMaxWidth((0));
+    }
   }, [outfit]);
 
   return (
     <ListWrapper>
-      <ListTitle>Your Outfit</ListTitle>
+      {/* <ListTitle>Your Outfit</ListTitle> */}
       {scrollValue > 0 && <LeftButton onClick={() => { handleArrowClick('left'); }}>&#10094;</LeftButton>}
       <ListCarousel ref={divRef}>
```

Comment:
I was thinking about this a while ago, maybe create a reusable carousel component for this section?